artifact/artifact/v1alpha/artifact_public_service.proto

```diff
@@ -42,6 +42,8 @@ service ArtifactPublicService {
     option (google.api.method_visibility).restriction = "INTERNAL";
   }
 
+  // TODO improve public endpoint definitions with title and description.
```

Comment:
cc @Yougigun you're welcome to contribute to this PR or tackle this in a separate one.